setup.py

```diff
@@ -17,7 +17,7 @@
 
 setup(
     name='sctools',
-    version='0.1.4',
+    version='0.1.6',
```

Comment:
Why is this version `0.1.6` and not `0.1.5`?

Reply:
Because I need to start using the pypi test server instead of pushing things with bugs directly to pypi's main index. :)

Reply:
Ohh okay! What was the bug? Do you think additional tests could help or was it really minor like a typo?

Reply:
Yea.. I think there was a testing failure. I'll think about how to add them. 👍